sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/ObjectAnchorsConversionClientOptions.cs

```diff
@@ -70,11 +70,11 @@ public ObjectAnchorsConversionClientOptions(ServiceVersion version = ServiceVers
         public enum ServiceVersion
         {
             /// <summary>
-            /// Version 0.2-preview.0 of the Azure Object Anchors service.
+            /// Version 0.3-preview.0 of the Azure Object Anchors service.
             /// </summary>
 #pragma warning disable CA1707 // Identifiers should not contain underscores
 #pragma warning disable AZC0016 // Invalid ServiceVersion member name.
-            V0_2_preview_0 = 1,
+            V0_3_preview_0 = 1,
```

Comment:
>V0_3_preview_0 = 1, [](http://example.com/codeflow?start=12&length=19)

You're removing the old version. The intent of this enum was to make a new entry for each version of the service.